### ISSUE TITLE
Missing definition

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#224]
Playing a match against Q-Plus bridge I noticed this
0B26F6BE4ECAC2A4A2736F38C92F
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/192ae0e8-d073-47a7-89d3-ed101a806062)

So 3C has no definition.

It could be Natural, Michaels or asking for stopper.
Q-Plus is treating it as natural

It seems to be a problem for all jump cuebids.

I prefer it is asking for stopper (typical a solid suit), but 1C-3C as natural is OK

**Comment by EdwardPiwowar** [comment]
Like this?
 http://www.edmontonbridge.ca/Crosby_files/Coaches%20Corner/Jump%20Cue%20Bids.htm

**Comment by ThorvaldAagaard** [comment]
Yes, that could be fine. Perhaps we should check how WBridge6, Jack and Q-Plus Bridge is playing it

**Comment by ThorvaldAagaard** [comment]
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/e3704d5a-2342-4ddf-a5cf-82f5bcebb1b4)
